johnsonandjohnson/Bodiless-JS
# fix(bodiless-layouts) Change # of components in flow container / remove red border / remove tailwind nags

## Changes

- Changed number of flow container componets to 25 and adjust messaging.
- Add future tailwind deprecations/statuses to remove tailwind nags
- Removed red border on edit mode

## Test Instructions
* Test Site:
    * Do a npm run start and verify the tailwind nags are gone.
    * Bring up the flow container and verify only 25 components are showing and the new message presents.  
        * Note the new message on the 4 column is still unreadable and another ticket is coming to fix that.
    * Verify no red border around editor in Test Site or Bodiless Test in either edit/review mode.
    * Do a npm run build and verify the tailwind nags are gone.
* Repeat above test for Starter site, except it doesn’t have 25 components yet so you can’t test it there.

## Related Issues
